artifactory/commands/golang/go.go

```diff
@@ -122,6 +127,29 @@ func (gc *GoCommand) Run() error {
 		return err
 	}
 	if isCollectBuildInfo {
+		tempDirPath := ""
+		if isGoGetCommand := len(gc.goArg) > 0 && gc.goArg[0] == "get"; isGoGetCommand {
+			if len(gc.goArg) < 2 {
+				// Package name was not supply, invalid go get commend
+				return errorutils.CheckError(errors.New("invalid get command, package name is missing"))
+			}
+			rtDetails, err := resolverDetails.CreateArtAuthConfig()
+			tempDirPath, err = fileutils.CreateTempDir()
+			if err != nil {
+				return err
+			}
+			err = copyGoPackageFiles(tempDirPath, gc.goArg[1], gc.resolverParams.TargetRepo(), rtDetails)
+			if err != nil {
+				return err
+			}
+			// Cleanup the temp working directory
+			defer fileutils.RemoveTempDir(tempDirPath)
+		}
+		// The version is not necessary because we are collecting the dependencies only.
```

Comment:
The reader may not understand which version this comment is referring to, and how the comment related to the following code line.

Reply:
Those are the original lines that was moved from outside the If `isCollectBuildInfo` block. 
I deleted the comment.